### DISCUSSION TITLE
建议显示发布日期距今时间

### DISCUSSION THREAD

**Discussion by luohuayouyi12138** [the1812/Bilibili-Evolved#2295] [功能建议 / Ideas]
在"#viewbox_report > div > span:nth-child(3)"后显示距今时间。

**Comment by CKylinMC** [comment]
![image](https://user-images.githubusercontent.com/11648014/134919885-9587895b-6d0f-45e5-a8f7-5715565b15b2.png)

https://greasyfork.org/zh-CN/scripts/398655

自荐

**Reply by Geeyun-JY3** [reply]
你这好像是脚本加的投稿时间吧。人家要发布时间。

**Reply by CKylinMC** [reply]
可是一般能看到的好像也就只有投稿时间吧...

**Reply by Geeyun-JY3** [reply]
？你这功能不是针对你这脚本提供的投稿时间吗？这里的需求不是针对原本就显示的时间吗？
